test_file.rb

```
@@ -0,0 +1,23 @@
+def doStuff(input) # bad method name, camelCase
+  foo = 1
```

RuboCop (1.76.1)

[warning] Useless assignment to variable - `foo`.

(Lint/UselessAssignment)